test/onnx/test_fx_to_onnx_with_onnxruntime.py

```diff
@@ -370,12 +370,6 @@ def forward(self, x, y):
             DynamicAdd(), (x, y), additional_test_inputs=[((input_x, input_y),)]
         )
 
-    @pytorch_test_common.skip_dynamic_fx_test(
```

Comment:
I think you meant to remove these kind of decorators `@pytorch_test_common.skip_min_ort_version` instead of the ones you did in this PR?

Reply:
Actually, nvm...

Reply:
😨